client/src/initial/landingpagetanu/tanulanding.scss

```diff
@@ -0,0 +1,120 @@
+.page-container {
+  background: repeating-linear-gradient(35deg, #c572f8 0px 2px, #b954f8 0px 4px);
+
+  top: 0;
+  height: 100vh;
+}
+.title-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  padding-top: 8%;
+  .under-construction-text {
+    color: rgb(48, 47, 47);
+    font-style: bold;
+    text-align: center;
+    font-size: 80px;
+
+    background: repeating-linear-gradient(145deg, #ecf023 0px 8px, #f3f70d 0px 12px);
```

Comment:
for any colours make sure to use the colours defined in this file, `client\src\util\colors.jsx`, this is just to make sure that our colours follow the branding guidelines, 

you'll notice that there is an array of colours defined, since we don't have the darkmode enabled on the landing page components, just make sure to look at the first colour code in the array. for example...
`'--purple': ['#6f1e88', '#613480'],`
you can call it using `var(--purple)` and it will display `#6f1e88`